lms/static/sass/course/_info.scss

```diff
@@ -48,7 +48,7 @@ div.info-wrapper {
     width: 100%;
     display: block;
 
-    h1 {
+    h1,h3 {
```

Comment:
@andy-armstrong  with that, I know my html pages to modify to use class instead but what  about other pages using same h1 styles.